src/main/java/com/atomshermental/springbootnewsservice/NewsRepositoryMock.java

```diff
@@ -0,0 +1,32 @@
+package com.atomshermental.springbootnewsservice;
+
+import com.atomshermental.springbootnewsservice.Interfaces.NewsRepository;
+import com.atomshermental.springbootnewsservice.Objects.News;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+
+@RequiredArgsConstructor
+@Repository
+
+public class NewsRepositoryMock implements NewsRepository {
+
+    private final Map<Long, News> storage;
+
+    private Long max = 0L;
```

Comment:
max initial value in repository must be calculated based on an initial storage value